assignments/function-conversion.js

```diff
@@ -5,26 +5,42 @@
 // };
 // myFunction();
 
+let myFunction = () => console.log("Function Was Invoked!");
+myFunction();
+
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => param;
```

Comment:
small note, if there's only one param, you can leave out the parens.  Either way works.  Just additional syntax that is valid